pkg/vault/contracts/Vault.sol

```diff
@@ -629,12 +611,13 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
         LiquidityLocals memory locals;
         locals.numTokens = poolData.tokens.length;
         amountsInRaw = new uint256[](locals.numTokens);
-        uint256[] memory swapFeeAmountsScaled18;
+        // swapFeeAmounts shall store scaled18 amounts first, and then it will be reused to store raw amounts.
```

Comment:
```suggestion
        // `swapFeeAmounts` stores scaled18 amounts first, and is then reused to store raw amounts.
```
Would discourage "shall"; sounds like the bible (though I guess that's "shalt").